keyboards/z150_blackheart/keymaps/default/keymap.c

```diff
@@ -0,0 +1,73 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define ______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
Like @mechmerlin said, it's the same with KC_NO as XXXXXXX.

Lines 3-5 are not needed and can be safely deleted.